org.jcryptool.visual.extendedrsa/src/org/jcryptool/visual/extendedrsa/RSAKeySimple.java

```diff
@@ -0,0 +1,48 @@
+package org.jcryptool.visual.extendedrsa;
+
+import java.math.BigInteger;
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.TreeSet;
+
+public interface RSAKey {

```

Comment:
Build fails here
```
[ERROR] 	public interface RSAKey {
[ERROR] 	                 ^^^^^^
[ERROR] The public type RSAKey must be defined in its own file
```

Reply:
Oh, must have messed something up when rebasing the branch, will fix